certificate.tex

```diff
@@ -7,8 +7,13 @@ \chapter*{Certificate}
 adopted by its Board of Directors on \dateadopted.
 
 \vspace{0.4in}
+\noindent \rule{3.5in}{0.8pt}\\
+Signature of Secretary
+\hspace{1in}
+Date
 
-\rule{3in}{0.8pt}
+\vspace{0.4in}
+Affix seal of Beta WPI Housing Corporation $\rightarrow$
```

Comment:
What if we don't have a seal yet?

Reply:
One of the resolutions to pass during the meeting orders the Secretary to get a seal. The secretary can put the seal on it after they get one.